src/schemas/versionedSection.ts

```diff
@@ -0,0 +1,54 @@
+import gql from "graphql-tag";
+
+export const typeDefs = gql`
+  extend type Query {
+    "Get all of the VersionedSection for the specified Section ID"
+    sectionVersions(sectionId: Int!): [VersionedSection]
+    "Search for VersionedSection whose name contains the search term"
+    publishedSections(term: String!): [VersionedSection]
+  }
+
+  "Section version type"
+  enum SectionVersionType {
+    "Draft - saved state for internal review"
+    DRAFT
+    "Published - saved state for use when creating DMPs"
+    PUBLISHED
+  }
+
+  "A snapshot of a Section when it became published."
+  type VersionedSection {
+    "The unique identifer for the Object"
+    id: Int!
+    "ID of the parent VersionedTemplate"
+    versionedTemplateId: Int!
```

Comment:
I think this should be `versionedTemplate: VersionedTemplate!` instead. We will probably want to be able to get at the VersionedTemplate properties from here without making a separate call (for example the template name or the affiliation).

Reply:
@briri Ok. That makes sense. I will make that update